hypertools/plot/draw.py

```diff
@@ -42,6 +42,13 @@ def plot1D(data):
         fig, ax = plt.subplots()
         for i in range(n):
             ikwargs = kwargs_list[i]
+            if 'color' in ikwargs:
```

Comment:
Do we support R,G,B tuples as a means of specifying colors?  If so, I think this new code might break that functionality (it appears that way at first glance)

Reply:
Yes, I have to be careful here. I will check this part; shouldn't be very difficult.

Reply:
we do support RGB tuples (+alpha) to specify colors

Reply:
i dont think you need to modify the code here.  in the plot function if clustering is passed, the data are reshaped based on the number of clusters to be a list of length(n_clusters).  In the case of the mixture model, i think we want to reshape the data into the number of unique RGB colors after the weighted average occurs. If plotting this seems like it is too computationally intensive, we can bin the RGB colors into maybe 100 bins using `np.digitize`/

Reply:
As suggested by @andrewheusser Since we always reshape the data according to the number of clusters, before calling the `plot` method, I haven't modified the `plot` method.
Regarding the bins, detailed comments are in a separate thread.